pipe_segment/transform/read_source.py

```diff
@@ -1,59 +0,0 @@
-from apache_beam.io.gcp.bigquery_tools import BigQueryWrapper
```

Comment:
@jennvanosdel the file removed `pipe_segment/transform/read_source.py`, I'm seeing that is being used for main segment pipeline: https://github.com/GlobalFishingWatch/pipe-segment/blob/jenn.segment-identity-daily-fixes/pipe_segment/pipeline.py#L18
Used for [reading the messages](https://github.com/GlobalFishingWatch/pipe-segment/blob/jenn.segment-identity-daily-fixes/pipe_segment/pipeline.py#L70). Should be the import updated?